cli/command/service/scale.go

```diff
@@ -43,11 +54,15 @@ func scaleArgs(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func runScale(dockerCli *command.DockerCli, args []string) error {
+func runScale(dockerCli *command.DockerCli, flags *pflag.FlagSet, options *scaleOptions, args []string) error {
 	var errs []string
+	var serviceIDs []string
+	ctx := context.Background()
+
 	for _, arg := range args {
 		parts := strings.SplitN(arg, "=", 2)
 		serviceID, scaleStr := parts[0], parts[1]
+		serviceIDs = append(serviceIDs, serviceID)
```

Comment:
Do we still want to wait on a service if it failed to update?

Maybe we should only add the serviceID to this list if there is no error from `runServiceScale()` ?